src/tasks/scanPlugins.js

```diff
@@ -44,10 +44,11 @@ const createPluginCheck = strict => ([, packageInfo]) => (
   packageInfo.name && (!strict || (strict && packageInfo.name.startsWith(PLUGIN_PREFIX)))
 )
 
-const scanPluginsByPath = (pluginsPath, checker) => (
+const scanPluginsByPath = (pluginsPath, checker) =>
   glob.sync(pluginsPath)
     .map(pluginPath => [pluginPath, readPackageJson(pluginPath)])
     .filter(checker)
+    .filter(([pluginPath]) => !fs.lstatSync(pluginPath).isSymbolicLink())
```

Comment:
Perhaps u need to filter before reading `package.json`, because, search goes into `./node_modules/@rispa`